datahub_sap_hana/ingestion.py

```diff
@@ -278,6 +278,13 @@ def get_column_view_lineage_elements(
                     dataset=view,
                 )
 
+                # checks the casing for the downstream column based on what is in the db
+                downstream_table_metadata = get_table_schema(
+                    inspector, downstream.dataset.name, view.schema)
```

Comment:
this is pretty small, but the downstream wont change in each loop (or shouldn't) so it is probably safe to do this right before the loop instead. but it is otherwise :+1: 

the main benefit of doing it before the loop is that you aren't recreating the dictionary each time. It _is_ cached, but you have some reassignments etc happening that might as well be avoided.